src/runtime/components/overlays/Popover.vue

```diff
@@ -15,6 +15,10 @@
       </slot>
     </HPopoverButton>
 
+    <Transition v-if="overlay" appear v-bind="ui.overlay.transition">
+      <div v-if="(open !== undefined) ? open : headlessOpen" :class="[ui.overlay.base, ui.overlay.background]" />
```

Comment:
Would that make sense to use [`PopoverOverlay`](https://headlessui.com/vue/popover#adding-an-overlay) component here?

Reply:
That's what I thought, but it doesn't seem to work with `manual` mode. It's missing the `static` prop that their `PopoverPanel` component has which allows use to ignore their internally managed state